dev-db/etcd/files/etcd2.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=etcd2
+Conflicts=etcd.service
+
+[Service]
+User=etcd
+PermissionsStartOnly=true
```

Comment:
There isn't an ExecStartPre these days so this doesn't do anything as far as I know.